modules/concepts/hooks/use-hooks-on-modern-pages.md

```diff
@@ -247,7 +247,7 @@ public function hookDisplayDashboardToolbarIcons($params)
 
         $this->writeFile($productsXml, $filepath);
 
-        return $this->get('twig')->render('@Modules/Foo/download_link.twig', [
+        return $this->get('twig')->render('@Modules/Foo/views/download_link.twig', [
```

Comment:
If we modify line, I think we also need to modify line 264

Reply:
Certainly, I already recommended it in https://github.com/PrestaShop/docs/pull/1366/files.